docs/scos/dev/front-end-development/202307.0/oryx/oryx-localization.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Oryx: Localization"
+description: Localizations are decoupled from component implementations
+template: concept-topic-template
+last_updated: July 23, 2023
+---
+
+Localization is the part of internationalization (`i18n`) that's concerned with creating variations of the same application in different locales.
+
+While most localized content is managed in other systems, like product or content management systems, components often use small text fragments in the UI. The examples are static site labels or aria labels.
+
+To make sure all components can be customized and localized, Oryx doesn't provide _any_ hardcoded text. Instead, components use _tokens_ that function as a reference to a text. When there's no localized text available for such a reference, Oryx renders the token in a human-readable format.
+
+## Translation keys
+
+Translation keys, also knows as _i18n tokens_, are used to resolve localized text. The tokens are created with the following conventions in mind:
+
+- Tokens are written in English.
+- Tokens are written in kebab-case format—for example, `my-token`.
+- Tokens are organized by domains—for example, `cart.add-to-cart`.
+- Tokens support context variables, which are wrapped in angle brackets in camelCase format. For example, `cart.totals.<count>-items`.
+
+## Resolving translations from translation resources
+
+The localization of labels is driven by the current language and the translation key. Translations are typically provided as lazy-loaded resources next to the component implementation. Although, they can aslo be added as part of the static resources that are loaded in Oryx. If the resources are loaded externally, they can be provided by static JSON files or a third-party service through an API.
+
+Oryx uses the active application language to look up the available labels. When the language is `en`, the locales for English are resolved.
+
+If a language resource can be resolved, the translation key is evaluated against the available translations in the resource using all "parts" of the key. For example, if a translation key contains multiple parts, like `cart.increase`, you can provide a global translation for just the `increase` part or for `cart.increase`.
+
+## Auto-conversion of translation keys
+
+If a translation key doesn't match any of the translations, or if the i18n feature is not installed (which is the default behavior), the translation key is converted into a human-readable message.
+
+For example, if a token contains multiple parts, like `cart.increase`, you can provide a global translation for just the `increase` part or for `cart.increase`. This mechanism allows for a single translation of `increase` that might affect multiple components. This provides a consistent and convenient translation mechanism while remaining flexible to add specific localizations for some components.

```

Comment:
This paragraph doesn't fit here, should be in previous point?

Reply:
for me. it's about providing translations and how translation works, not about auto-conversion of translation tokens?